src/sentry/api/serializers/models/dashboard.py

```diff
@@ -171,6 +181,16 @@ def serialize(self, obj, attrs, user, **kwargs):
                 if obj.filters.get(camel_to_snake_case(key)) is not None:
                     data["filters"][key] = obj.filters[camel_to_snake_case(key)]
 
+            if obj.filters.get("release_id"):
+                data["filters"]["releaseData"] = [
+                    serialize(release, user, DashboardReleaseSerializer())
+                    for release in Release.objects.filter(
+                        id__in=filter(lambda id: id != "latest", obj.filters["release_id"])
```

Comment:
I had to because I was getting an error saying that the `id` value is expecting a number. It seems like filtering on id will convert strings to numbers, but `latest` can't be converted successfully

Reply:
Can we also add org id and/or projects to this query?

Reply:
Added org id to the query!